x-pack/filebeat/module/threatintel/malwarebazaar/config/config.yml

```diff
@@ -5,19 +5,14 @@ interval: {{ .interval }}
 
 request.method: POST
 {{ if .ssl }}
-
 request.ssl: {{ .ssl | tojson }}
 {{ end }}
 {{ if .proxy_url }}
 request.proxy_url: {{ .proxy_url }}
 {{ end }}
 request.url: {{ .url }}
-#request.encode_as: application/x-www-form-encoded
-
+request.encode_as: application/x-www-form-urlencoded
```

Comment:
I am a bit unsure if this was commented out on purpose already, though I have tested the package/fleet version with this option and it seems fine.

Reply:
it had a typo in the comment (`form-encoded` vs `form-urlencoded`) so that is likely what it made it fail, this should work as adding the header manually

Reply:
Gotcha!